test.js

```diff
@@ -3,22 +3,22 @@
 var test = require('tape').test
 var persistence = require('./')
 var abs = require('aedes-persistence/abstract')
-var levelup = require('levelup')
-var memdown = require('memdown')
+var level = require('level')
+var tempy = require('tempy')
 
-function memdb () {
-  return levelup(memdown())
+function leveldb () {
+  return level(tempy.directory())
```

Comment:
`tempy.directory()` is very useful, we use it in `leveldown` and other modules.

Reply:
:+1: